src/lib.rs

```diff
@@ -1,37 +1,304 @@
+//!                 .run("project")?;
+//!
+//!             println!("Built at {}", dist.dist_dir.display());
+//!
+//!         }
+//!         Command::Watch(watch) => {
+//!             let mut command = process::Command::new("cargo");
+//!             command.args(["xtask", "dist"]);
+//!
+//!             println!("Starting to watch");
+//!             watch.exclude_workspace_path("dist").run(command)?;
+//!         }
+//!         Command::Serve(mut dev_server) => {
+//!             let mut command = process::Command::new("cargo");
+//!             command.args(["xtask", "dist"]);
+//!
+//!             dev_server.watch = dev_server.watch.exclude_workspace_path("dist");
+//!
+//!             println!("Starting the dev server");
+//!             dev_server.command(command).start("dist")?;
+//!         }
+//!     }
+//!
+//!     Ok(())
+//! }
+//! ```
+//!
+//! You can find further information for each type at their documentation level.
+//!
+//! # Examples
+//!
+//! * A basic implementation could look like this:
+//!     ```rust,no_run
+//!     use std::process;
+//!     use xtask_wasm::{anyhow::Result, clap};
+//!
+//!     #[derive(clap::Parser)]
+//!     struct Opt {
+//!         #[clap(subcommand)]
+//!         cmd: Command,
+//!     }
+//!
+//!     #[derive(clap::Parser)]
+//!     enum Command {
+//!         Dist(xtask_wasm::Dist),
+//!         Watch(xtask_wasm::Watch),
+//!         Serve(xtask_wasm::DevServer),
+//!     }
+//!
+//!
+//!     fn main() -> Result<()> {
+//!         let opt: Opt = clap::Parser::parse();
+//!
+//!         match opt.cmd {
+//!             Command::Dist(dist) => {
+//!                 let dist = dist
+//!                     .dist_dir_path("dist")
+//!                     .static_dir_path("project/static")
+//!                     .app_name("project")
+//!                     .run_in_workspace(true)
+//!                     .run("project")?;
+//!
+//!                 log::info!("Built at {}", dist.dist_dir.display());
+//!
+//!             }
+//!             Command::Watch(watch) => {
+//!                 let mut command = process::Command::new("cargo");
+//!                 command.args(["xtask", "dist"]);
+//!
+//!                 log::info!("Starting to watch");
+//!                 watch.exclude_workspace_path("dist").run(command)?;
+//!             }
+//!             Command::Serve(mut dev_server) => {
+//!                 let mut command = process::Command::new("cargo");
+//!                 command.args(["xtask", "dist"]);
+//!
+//!                 log::info!("Starting the dev server");
+//!                 dev_server.command(command).start("dist")?;
+//!             }
+//!         }
+//!
+//!         Ok(())
+//!     }
+//!     ```
+//!
+//! * [`examples/demo`](https://github.com/rustminded/xtask-wasm/tree/main/examples/demo)
+//!     provides an implementation of xtask-wasm to build the `webapp` package, an
+//!     `hello world` app using [Yew](https://yew.rs/). This example demonstrate a
```

Comment:
backticks are normally for technical values of some sort... or code.

I demonstrate
You demonstrate
He/She/It demonstrate**s**

It's so much easier than French. You can do it! I believe in you!

```suggestion
//!     "hello world" app using [Yew](https://yew.rs/). This example demonstrates a
```